### DISCUSSION TITLE
Ender 3 Pro Usage?

### DISCUSSION THREAD

**Discussion by sysrazor** [Jyers/Marlin#442] [Q&A] [answered]
I have an Ender 3 v2 4.2.2. w/ bltouch and am getting ready to use your 3.0.0 firmware; I also have a newer Ender 3 pro with the 4.2.2 board w/ bltouch and am curious if anyone has attempted to use Jyers firmware with it.

**Comment by smooreace** [comment, marked as answer]
Should all be the same if both have the same 4.2.2 bourd

**Comment by jwinok** [comment]
Were you able to get it to work? My nephew has the same setup but hasn't been able to get it working because of the difference in displays. He said the 3 pro's display is blank. Any tips for how to get it going?

**Reply by sysrazor** [reply]
No I had the same experience today as well.

**Comment by jwinok** [comment]
Does anyone know whether grabbing the jyers source code and then making the changes made in this link: https://crosslink.io/2021/02/19/ender-3-v2-im-switching-back-to-the-12864-display-the-why-and-how/
where the guy switched his E3V2 display back to the E3pro display would result in the jyers firmware  running on a E3Pro with 4.2.2 board?  I've never compiled FW but my little nephew is disappointed his printer doesn't work anymore.

**Reply by t3chguy** [reply]
The whole point of Jyers source code is improved support for the E3V2 display, everything else is just from latest bugfix Marlin. Running it with E3Pro display will not give you any benefits.

**Reply by cruzer619** [reply]
True if you know how to compile the firmware to get the features your after..   And like most.. we are not that "savy" to do so.. well at least i'm not.. hahaha.. so for us un-savy persons.. this would help.  I'm after the same thing as just picked up a 2nd printer E3Pro.

**Reply by t3chguy** [reply]
Well you can take the config from this repo, switch the bits about the screen and use the latest Marlin codebase. This repo is just 

Marlin modified with better E3V2 screen support + some config files + automatic builds every release

Only the config files have any benefit to anyone not using an E3V2 screen.

**Reply by jwinok** [reply]
Thanks T3chguy for the reply.
My nephew ended up with a 4.2.2 board and has a BLTouch and hasn't been able to use it so while I understand that jyers FW is meant to enhance the E3V2 display experience (which was terrible) I don't have the knowledge to set up a config file that works like Mr. Meyers has done.  I can maybe figure out how to use these config files, switch the display info and compile it with Marlin.  I would like to be better at the firmware modifications but I don't have any background that would ease the learning curve and I have been putting all of my spare learning time into learning to 3d model so I can make 3d printing useful to me.

**Reply by cruzer619** [reply]
I guess I'll be more direct... Can someone compile this for us 8bit minded persons for the Ender 3 pro.  Will be greatly appreciated ... I'll even throw in a beer :P .
